sdk/healthinsights/health-insights-radiologyinsights-rest/eslint.config.mjs

```diff
@@ -0,0 +1,14 @@
+import azsdkEslint from "@azure/eslint-plugin-azure-sdk";
+
+export default azsdkEslint.config([
+  {
+    rules: {
```

Comment:
What was the justification for changing these rules to warn or turning them off? Or was this from codegen / automation added?